Point_set_processing_3/include/CGAL/Point_with_normal_3.h

```diff
@@ -147,7 +147,8 @@ struct Normal_of_point_with_normal_pmap
   typedef Normal_of_point_with_normal_pmap<Gt> Self;
   /// \name Put/get free functions
   /// @{
-  friend reference get(const Self&,const key_type& k) {return k.normal();}
+  friend const reference get(const Self&,const key_type& k) {return k.normal();}
```

Comment:
I must miss something because I don't see the problem. Are you talking of `const reference` vs `const value_type&`?